docs/diagrams/DeleteOrderActivityDiagram.puml

```diff
@@ -0,0 +1,16 @@
+@startuml
+start
+:User executes delete order command;
+
+if () then ([!isOrderFinished])
```

Comment:
Can it match the sequence diagram? Also is it acceptable to use `!` as not in activity diagram?
```suggestion
if () then ([order is not finished])
```